src/accounts/context/userAccount.tsx

```diff
@@ -83,32 +86,25 @@ export const UserAccountProvider: FC<Props> = React.memo(({children}) => {
 
   const handleGetAccounts = useCallback(async () => {
     try {
-      const resp = await getAccounts({})
-      if (resp.status !== 200) {
-        // set user account status to error;...TODO
-        throw new Error(resp.data.message)
+      const accounts = await getUserAccounts()
+
+      setUserAccounts(accounts)
+      const defaultAcctArray = accounts.find(line => line.isDefault)
```

Comment:
This isn't absolutely necessary, but one suggestion I have is that `line` could also be more semantic - it might be clearer if the element were `acct` or `account`.

Reply:
i was just moving the existing code. but good catch. i will rename it.